### ISSUE TITLE
Add 10.6 Glyph Stats

### ISSUE THREAD

**Issue by Arany** [joakibj/tswcalc#44]
With Manufactury Breached, there are now 10.9 items with 10.6 glyphs.
Would it be possible to add the glyphs to the calculation?

**Comment by Xeio** [comment]
Biggest problem is we are missing glyph values for the vast majority of 10.6 combinations.

Granted, several of the useful ones like hit/pen/crit (for 4/4 glyphs, for some slots) could theoretically be added.

**Comment by Arany** [comment]
Maybe, until the stats are available, grey out the options to do the combinations that are unavailable? or make them "0". So you can already at calculate with the ones that are known?

**Comment by Inkraja** [comment]
The Game only contains 10.0-10.5 and 11 Glpyhs, so their values are known. 10.6-10.9 are not inside only as bound glyp on the mfa/b items, like the raid items.

**Comment by Arany** [comment]
Then maybe the question is, if there is a way to add those items? Maybe In line with the NYR + Woodcutter items. At least currently they are THE most end-gamey items that you can get.
At some point I guess there is no more room for more checkboxes there and it gets more and more complicated..

**Comment by Inkraja** [comment]
Getting more checkboxes and switch them do a dropdown menu shouldnt a big problem. The only important questions is, if @joakibj want to do that :smile_cat:

**Comment by Xeio** [comment]
I'd agree moving the checkboxe options into the dropdown is probably the best option, but it also requires rewriting a lot of tests and moving signet/glyph logic into the data for the dropdown options. We'd probably want to keep the list of gear in there somewhat trimmed as well, mostly to gear that can't be made from equivalent customs.

Personally I'm probably not particularly keen on doing any of that till we have new NM gear to account for (11.0 support is one of those needed things too). Short term the PvP signets will need to be added as well, though they haven't been finalized yet so...

**Comment by joakibj** [comment]
Yeah dropdowns are probably the more scalable solution in the current GUI. e.g. select by role+slot+item name instead of role+slot+raid combination.

But to be clear, these items haven't started dropping yet? Just datamined from TestLive?
Also if anyone wants to dig around in the .rdbdata with for example tswrdb then I bet the actual items are defined somewhere in there. So they can be programmatically extracted like any other sane item database.

Regarding tests, they are important for protecting against regression, but user value comes first.

**Comment by joakibj** [comment]
Alright, any list out there with the Manufactory:Breached loot ?

**Comment by Xeio** [comment]
I don't think anyone has compiled a list of the 10.9 loot, that would probably be work we have to do.

I have 4-5 of the pieces myself, could possibly ping cabalies for a lot of the others. Might be easier to ask for help getting a full list on the main forums though.

Oh, and I forgot there are 10.9.6 variants of every weapon too (though they all have a 10.6 Hit Glyph, so it's mostly which signet is built into each that would be needed).

**Comment by Inkraja** [comment]
PVP Signets are here (coming live, soon): https://docs.google.com/spreadsheets/d/11AOygXcnzmZHr9bdLmntGEUgCZWA9t5EdKGZYzYuNns/edit?usp=sharing
NM Raid Items, i have a similar Sheet, but i will never ever share it in public, and till the first NYR nm i try to keep the secrets as long as possible. ;) 

The mfb Items have also signets. For example 10.8 AR with hit glyph and blue abuse, 10.9 AR with hit glyph and purple Abuse.
The Head DPS Piece is similar 10.8 with Pen glyph and blue Breaching, 10.9 with Hit glyph (yes 10.8 and 10.9 have a different glyph) and purple Breaching.

Xeio i can send u the ingame datamining script, but u need to know the item id's.

**Comment by Inkraja** [comment]
https://docs.google.com/spreadsheets/d/1PEhLIOrDC2mcYRQpsupNeaEEnAell22ORyikswcMu4A/edit?usp=sharing
10.9 Items have purple signets, 10.8 blue one

**Comment by Xeio** [comment]
So a minor update on QL11 glyphs:

Currently 1/3 and 3/1 splits do not exist (So glyphs like 3 Crit Power, 1 Pen can't be made). I found this oddity in the data mining, but I confirmed it with Headshoty that the crafting window does not allow preview of those variants.

Have most of the stats collected and most of the code work done, so I basically just need to find time to put everything into the data file (and add handling to prevent selecting 1/3,3/1 variants).
